lib/containers/utils.pm

```diff
@@ -155,12 +155,14 @@ sub basic_container_tests {
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
     # Containers' state can be saved to a docker image
-    if (script_run("$runtime container exec $container_name zypper -n in curl", 300)) {
+    my $ret = script_run("$runtime container exec $container_name zypper -n in curl", 200);
```

Comment:
why decreasing the timeout here?

Reply:
Should we use script_retry here? Similar to  https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13542

Reply:
Sorry, I increased it to 420 now.

I don't think we need `script_retry()`.